phpBB/language/en/email/newtopic_notify.txt

```diff
@@ -1,18 +1,16 @@
-Subject: New topic notification - "{FORUM_NAME}"
+Subject: Topic in "{FORUM_NAME}"
```

Comment:
```suggestion
Subject: New topic in "{FORUM_NAME}"
```

Reply:
But isn't it quite obvious, that the topic and reply is new? If not, the would be a need for a notification mail at all.

Reply:
The other subjects have a word indicating what action was taken. `Topic in "Help forum"` does not have one so it doesn't tell the user anything about what actually happened, I'd have to guess or know already.  Can also be `Topic created`.

Reply:
Well then "new" is better, imo, because shorter and clear enough :)